lib/features/manage_account/presentation/menu/settings/settings_view.dart

```diff
@@ -217,7 +242,9 @@ class SettingsView extends GetWidget<SettingsController> {
           }
         case AccountMenuItem.forward:
           if (controller.manageAccountDashboardController.isForwardCapabilitySupported) {
-            return ForwardView();
+            return SafeArea(
+              top: false,
+              child: ForwardView());
```

Comment:
why we need to add `SafeArea` like that?

Reply:
We need horizontal padding for landscape mobile.
 
<table>
 <tr>
    <th>Before</th>
    <th>After</th>
  </tr>
  <tr>
    <td><img src="https://github.com/linagora/tmail-flutter/assets/80730648/b98fa938-b704-4f88-8f90-0ba38d00a4e3" /></td>
    <td><img src="https://github.com/linagora/tmail-flutter/assets/80730648/139512b0-4ebb-414c-b5a5-56bd4ec416ab" /></td>
  </tr>
</table>